var/spack/repos/builtin/packages/amdfftw/package.py

```diff
@@ -0,0 +1,210 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+import os
+import os.path
+import llnl.util.lang
+from spack import *
+from spack.pkg.builtin.fftw import FftwBase
+
+
+class Amdfftw(FftwBase):
+    """AMD Optimized FFTW.
+
+    FFTW is a comprehensive collection of fast C routines for
+    computing the Discrete Fourier Transform (DFT) and various special
+    cases thereof. It is an open-source implementation of the Fast
+    Fourier transform algorithm. It can compute transforms of real and
+    complex-values arrays of arbitrary size and dimension.
+    AMD Optimized FFTW is the optimized FFTW implementation targeted
+    for AMD CPUs.
+    """
```

Comment:
I would rather write something like this for the docstring:
```suggestion
    """FFTW (AMD optimized version) is a comprehensive collection of fast C routines for
    computing the Discrete Fourier Transform (DFT) and various special
    cases thereof. 
    
    It is an open-source implementation of the Fast
    Fourier transform algorithm. It can compute transforms of real and
    complex-values arrays of arbitrary size and dimension.
    AMD Optimized FFTW is the optimized FFTW implementation targeted
    for AMD CPUs.
    """
```
The rationale is to have a short but descriptive sentence as a first paragraph, and a longer description in a separate paragraph. The first sentence is extracted by Spack to be used whenever a succinct description of the package is needed

Reply:
Incorporated spack community review comments under - https://github.com/spack/spack/pull/18155